.github/workflows/coverity.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  schedule:
+    - cron:  '0 4 * * 6'
+
+env:
+  CI_MAKE: make -f .travis.mk
+
+jobs:
+  coverity:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+    container:
+      image: ghcr.io/avtikhon/tarantool/testing/debian-buster:latest
+
+    steps:
+      - uses: fkirc/skip-duplicate-actions@master
```

Comment:
It seems like we need to use [this](https://github.com/fkirc/skip-duplicate-actions#skip-concurrent-workflow-runs) feature of skip-duplicate not to trigger twice on PR and Push

Reply:
Right, found that entire jobs skip is needed in our way [1]

[1] - https://github.com/fkirc/skip-duplicate-actions#option-1-skip-entire-jobs